### ISSUE TITLE
TypeError in Hyrax::Stats#file

### ISSUE THREAD

**Issue by christhepianist** [samvera/hyku#3048]
<img width="879" height="973" alt="Image" src="https://github.com/user-attachments/assets/530cfe2b-366e-4166-997f-572e86eab8b3" />

Hyku version: 7.1
vpat.hykudemo.org

Descriptive summary
When testing for accessibility, we used SortSite to test for WCAG 2.1 AA compliance. This Rails error was seen on the following pages:
https://vpat.hykudemo.org/files/6980c798-dfb3-4394-87bb-72ff161c1fc7/stats?locale=de
https://vpat.hykudemo.org/files/6980c798-dfb3-4394-87bb-72ff161c1fc7/stats?locale=es
https://vpat.hykudemo.org/files/6980c798-dfb3-4394-87bb-72ff161c1fc7/stats?locale=fr
https://vpat.hykudemo.org/files/6980c798-dfb3-4394-87bb-72ff161c1fc7/stats?locale=it
https://vpat.hykudemo.org/files/6980c798-dfb3-4394-87bb-72ff161c1fc7/stats?locale=pt-BR
https://vpat.hykudemo.org/files/6980c798-dfb3-4394-87bb-72ff161c1fc7/stats?locale=zh
https://vpat.hykudemo.org/files/6980c798-dfb3-4394-87bb-72ff161c1fc7/stats?locale=en
https://vpat.hykudemo.org/files/b2e62b84-5490-4bb5-acd1-96ec6b64b5e3/stats?locale=de
https://vpat.hykudemo.org/files/b2e62b84-5490-4bb5-acd1-96ec6b64b5e3/stats?locale=es
https://vpat.hykudemo.org/files/b2e62b84-5490-4bb5-acd1-96ec6b64b5e3/stats?locale=fr
https://vpat.hykudemo.org/files/b2e62b84-5490-4bb5-acd1-96ec6b64b5e3/stats?locale=it
https://vpat.hykudemo.org/files/b2e62b84-5490-4bb5-acd1-96ec6b64b5e3/stats?locale=pt-BR
https://vpat.hykudemo.org/files/b2e62b84-5490-4bb5-acd1-96ec6b64b5e3/stats?locale=zh
https://vpat.hykudemo.org/files/b2e62b84-5490-4bb5-acd1-96ec6b64b5e3/stats?locale=en
https://vpat.hykudemo.org/files/ab6249c5-5170-456c-8368-9a72b4075747/stats?locale=de
https://vpat.hykudemo.org/files/ab6249c5-5170-456c-8368-9a72b4075747/stats?locale=es
https://vpat.hykudemo.org/files/ab6249c5-5170-456c-8368-9a72b4075747/stats?locale=fr
https://vpat.hykudemo.org/files/ab6249c5-5170-456c-8368-9a72b4075747/stats?locale=it
https://vpat.hykudemo.org/files/ab6249c5-5170-456c-8368-9a72b4075747/stats?locale=pt-BR
https://vpat.hykudemo.org/files/ab6249c5-5170-456c-8368-9a72b4075747/stats?locale=zh
https://vpat.hykudemo.org/files/ab6249c5-5170-456c-8368-9a72b4075747/stats?locale=en
https://vpat.hykudemo.org/files/740fa080-26dd-4cdc-96bd-ec7feee547a5/stats?locale=de
https://vpat.hykudemo.org/files/740fa080-26dd-4cdc-96bd-ec7feee547a5/stats?locale=es
https://vpat.hykudemo.org/files/740fa080-26dd-4cdc-96bd-ec7feee547a5/stats?locale=fr
https://vpat.hykudemo.org/files/740fa080-26dd-4cdc-96bd-ec7feee547a5/stats?locale=it
https://vpat.hykudemo.org/files/740fa080-26dd-4cdc-96bd-ec7feee547a5/stats?locale=pt-BR
https://vpat.hykudemo.org/files/740fa080-26dd-4cdc-96bd-ec7feee547a5/stats?locale=zh
https://vpat.hykudemo.org/files/740fa080-26dd-4cdc-96bd-ec7feee547a5/stats?locale=en
https://vpat.hykudemo.org/files/c9bd9ecf-1872-4b53-9675-a4496c5067f5/stats?locale=de
https://vpat.hykudemo.org/files/c9bd9ecf-1872-4b53-9675-a4496c5067f5/stats?locale=es
https://vpat.hykudemo.org/files/c9bd9ecf-1872-4b53-9675-a4496c5067f5/stats?locale=fr
https://vpat.hykudemo.org/files/c9bd9ecf-1872-4b53-9675-a4496c5067f5/stats?locale=it
https://vpat.hykudemo.org/files/c9bd9ecf-1872-4b53-9675-a4496c5067f5/stats?locale=pt-BR
https://vpat.hykudemo.org/files/c9bd9ecf-1872-4b53-9675-a4496c5067f5/stats?locale=zh
https://vpat.hykudemo.org/files/c9bd9ecf-1872-4b53-9675-a4496c5067f5/stats?locale=en
https://vpat.hykudemo.org/files/5584dabf-bb84-4b9e-a75d-3af4d6798ec9/stats?locale=de
https://vpat.hykudemo.org/files/5584dabf-bb84-4b9e-a75d-3af4d6798ec9/stats?locale=es
https://vpat.hykudemo.org/files/5584dabf-bb84-4b9e-a75d-3af4d6798ec9/stats?locale=fr
https://vpat.hykudemo.org/files/5584dabf-bb84-4b9e-a75d-3af4d6798ec9/stats?locale=it
https://vpat.hykudemo.org/files/5584dabf-bb84-4b9e-a75d-3af4d6798ec9/stats?locale=pt-BR
https://vpat.hykudemo.org/files/5584dabf-bb84-4b9e-a75d-3af4d6798ec9/stats?locale=zh
https://vpat.hykudemo.org/files/5584dabf-bb84-4b9e-a75d-3af4d6798ec9/stats?locale=en

Steps to reproduce the behavior in User Interface (UI)
* The site must be tested again with SortSite

Acceptance Criteria/Expected Behavior
* SortSite no longer throws this error

Related work
* Slack conversation with @maxkadel
